app/controllers/items/partial_undesignate.js

```diff
@@ -13,57 +15,88 @@ export default Ember.Controller.extend({
   partialUndesignateBackLinkpath: Ember.computed.localStorage(),
 
   init() {
-    this.set('promptUserPopUp', false);
+    this.set("promptUserPopUp", false);
+  },
+
+  cannotDispatchItems(designation) {
```

Comment:
Is this criteria is same as dispatching from item details page? 

Just confirming :)

Reply:
Yes